src/components/apiComponents/languageSelector.jsx

```diff
@@ -0,0 +1,46 @@
+import { Icon } from '@/components/icons'
+
+const languages = [
+  { name: 'javascript', icon: 'JavaScriptIcon' },
+  { name: 'python', icon: 'PythonIcon' },
+  { name: 'ruby', icon: 'RubyIcon' },
+  { name: 'go', icon: 'GoIcon' },
+  { name: 'java', icon: 'JavaIcon' },
+  { name: 'csharp', icon: 'CSharpIcon' },
+  { name: 'swift', icon: 'SwiftIcon' },
+  { name: 'kotlin', icon: 'KotlinIcon' },
+  { name: 'php', icon: 'PhpIcon' },
+  // { name: 'typescript', icon: 'TypescriptIcon' },
+  { name: 'shell', icon: 'ShellIcon' },
```

Comment:
How is 'shell' different than 'curl'?  I don't see 'shell' doing anything on the rendering right now.

Reply:
Yeah I can ditch shell.